### ISSUE TITLE
Consider rewriting the documentation

### ISSUE THREAD

**Issue by marjoforcado** [Innologica/vue2-daterange-picker#103]
The documentation is a bit hard to follow. Consider adding a simple code snippet for easier installation

**Comment by nkostadinov** [comment]
I know about that and will definetly do it. I've put a lot on my list but I have zero time these months.

The documentation will be the first thing to change. There is an example in the web page: https://innologica.github.io/vue2-daterange-picker/#usage

I know it can be better but still...

**Comment by nkostadinov** [comment]
New documentation is on the way. More sample and use cases and all props well documented.

**Comment by sourcenux** [comment]
I would like to bump this issue. I would be grateful for a updated documentation 👍. You may put the source code of your current documentation on github as well. 

Thanks!
